media/css/core.css

```diff
@@ -621,6 +621,14 @@ p.build-missing { font-size: .8em; color: #9d9a55; margin: 0 0 3px; }
   color: #5a5;
 }
 
+a.notification-action {
+    text-decoration: none;
+}
+
+.notification-action > .icon.close:before {
+    font-family: FontAwesome;
+    content: "\f057";
+}
```

Comment:
Fontawesome is not used on .com unfortunately, this would need separate attention or perhaps is better as text